File: 2219-maximum-number-of-words-found-in-sentences/2219-maximum-number-of-words-found-in-sentences.c
int mostWordsFound(char** sentences, int sentencesSize) {
    int max=0;
    for( int i = 0;i < sentencesSize;  i++){
        int c=1;
        for(int j = 0;sentences[i][j] != '\0'; j++){
        if(sentences[i][j]==' '){
            c++;
        }
        }
        if(c>max)max=c;
    }
    return max;
}

File: 2219-maximum-number-of-words-found-in-sentences/2219-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int l = 0;
        int max = 0;
        for(int i = 0 ; i < sentences.length;i++){
            int sum = 0;
         for(int j = 0 ; j < sentences[i].length();j++){
            if(sentences[i].charAt(j)!=' ') l =1 ;
            else{
                sum++;
                l = 0;
            }
         }
          if(sum > max ) max = sum;
        }
        return max+1;
    }
}